tpgtools/templates/resource.go.tmpl

```diff
@@ -243,7 +243,16 @@ func resource{{$.PathType}}Create(d *schema.ResourceData, meta interface{}) erro
 	if err != nil {
 		return err
 	}
-	client := Create{{$.ProductType}}Client(config, userAgent, "")
+	{{- if $.HasProject }}
```

Comment:
nit: template directives should be dedented by one, for consistency with other directives in this file

Reply:
dedented